docs/troubleshooting/repository-scripts.md

```diff
@@ -0,0 +1,71 @@
+   1. If **all** timestamps are more than 6 hours old, something may be wrong with cron or its mash entries:
+      1. If possible, verify that cron is running and that the cron entries for mash are still present; if not, try to restore things
+      1. Otherwise, create a GOC ticket with a subject like “Repo update logs are too old on repo**N**” and with relevant details in the body
+      1. Assign the ticket to “Software Support (Triage)”
+      1. Email Carl Edquist <edquist@cs.wisc.edu> and CC Tim Cartwright <cat@cs.wisc.edu> with the ticket URL
+
+Troubleshooting and Mitigation
+------------------------------
+<style type="text/css">
+  em.pgid { color: red; font-weight: bold; }
+  em.start { color: blue; font-weight: bold; }
+</style>
+
+### Identifying and fixing a hung mash process
+
+If a mash update process hangs, all future invocations from cron of the mash scripts will exit without taking action because of the hung process. Thus, it is important to identify and remove any hung processes so that future updates can proceed. Use the procedure below to remove any hung mash processes; doing so is safe in that it will not adversely affect the Yum repositories being served from the host.
+
+1. On the web listing of log files (see above), click on the =update_all_repos.err= link
+1. In the error log file, look for messages such as:  
+
+        :::file
+        Wed Jan 20 18:10:02 UTC 2016: **Can't acquire lock, is update_all_repos.sh already running?**  
+ 
+    This message indicates that the most recent update attempt quit early due to the presence of a lock file, most likely from a hung *mash* process.
+ 
+1. Log on to the host as *root* or with your regular user account
+1. Look for *mash* processes:  
+   
+        :::console
+        root@host # ps -C mash -o pid,ppid,pgid,start,command
+        PID  PPID  PGID STARTED COMMAND
+        24551 2454923455 Jan 15 /usr/bin/python /usr/bin/mash osg-3.1-el5-release -o
+        24552 2455123455 Jan 15 /usr/bin/python /usr/bin/mash osg-3.1-el5-release -o
+   
+1. If there are *mash* processes that started on a previous date or more than 2 hours ago, it is best to remove their corresponding process groups (PGID, highlighted above):  
+    
+        :::console
+        root@host # kill -TERM -23455
+       
+    Then verify that the old processes are gone using the same *ps* command as above:
+
+        :::console  
+        root@host # ps -C mash -o pid,ppid,pgid,start,command
+        PID  PPID  PGID  STARTED COMMAND
+
+1. If any part of this process does not look or work as expected:
+   1. Create a GOC ticket with a subject like “Problems with mash on repo<strong>N</strong>” and with relevant details in the body
+   1. Assign the ticket to “Software Support (Triage)”
+   1. Email Carl Edquist <edquist@cs.wisc.edu> and CC Tim Cartwright <cat@cs.wisc.edu> with the ticket URL
```

Comment:
Replace this with the same ticket creation text from above, after it's been fixed